nmtwizard/preprocess/tu.py

```diff
@@ -70,7 +70,7 @@ def align(self, src_tok, tgt_tok):
             for src_tok_part, tgt_tok_part in zip(src_tok, tgt_tok):
                 align_result = self.aligner.align(src_tok_part, tgt_tok_part)
                 # TODO : write fwd and bwd probs
-                alignments.append(set(align_result["alignments"]))
+                alignments.append(align_result["alignments"])
```

Comment:
I think the module `systran_align` will keep returning a list. So we should probably keep the conversion.

Reply:
Yes, you are right again. Sorry, I'm starting to loose track a bit.